Makefile

```diff
@@ -90,6 +93,10 @@ ifneq ($(wildcard /.dockerenv), )
 	go test -check.v -check.timeout=$(TEST_TIMEOUT) timeout=$(TEST_SUITE_TIMEOUT) $(TESTFLAGS) github.com/docker/containerd/integration-test
```

Comment:
It doesnt buy us much at this point, since there's one trivial workload running on a extremely thin image. Once we have more interesting workloads running on a variety of of rootfs bundles, we can add it to the test rule. WDYT?